assests/javascript/index.js

```diff
@@ -1 +1,225 @@
-//
+const apiKey = "8cf248a4fd398e2908cf52a9f187375b";
+const mainSection = document.getElementById("main");
+
+let city = {};
+let curentWeather = {};
+let dailyForecastingWeatherList = {};
```

Comment:
move this to `getWeather` function scope.

Reply:
ok thankssss